index.html

```diff
@@ -2653,7 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
-          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
+          <li><a>Trim</a>(<var>value</var>) and convert <a>to ASCII lowercase</a></li>
```

Comment:
What you propose is what I was thinking, but I searched the spec for "lower" and found:
* Three instances of: `to lower-case` (no `<a>` tag)
* Two instances of:  `to ASCII lowercase` (with `<a>` tag)

Do you know if there's any reason that all of five of these should not just follow your suggestion?